src/Compilers/VisualBasic/Portable/Binding/InitializerSemanticModel.vb

```diff
@@ -87,6 +87,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
         End Function
 
+        Private Iterator Function GetInitializedFieldsOrProperties(binder As Binder) As IEnumerable(Of Symbol)
```

Comment:
>GetInitializedFieldsOrProperties [](start = 34, length = 32)

It feels like we can do without this method.

Reply:
Can this method simply return an `ImmutableArray(Of TSymbol)`:
`Function GetInitializedFieldsOrProperties(Of TSymbol)(binder As Binder) As ImmutableArray(Of TSymbol)`?